### ISSUE TITLE
"Your server isn't responding to some requests": "requests" text not visible on light theme

### ISSUE THREAD

**Issue by anoadragon453** [element-hq/element-web#33494]
### Steps to reproduce

1. Go offline/disconnect from the homeserver
2. Change notification settings for a room
3. See the "your server isn't responding to some requests" notice pop up

### Outcome

When using the light theme, the "requests" link wasn't visible:

<img width="594" height="240" alt="Image" src="https://github.com/user-attachments/assets/de39f49d-726c-433a-86ec-c5a8cfac5f02" />

---

<img width="576" height="136" alt="Image" src="https://github.com/user-attachments/assets/304d152b-a238-4cb5-8dc2-0f2d0d3f03af" />

It's fine in dark mode:

<img width="576" height="136" alt="Image" src="https://github.com/user-attachments/assets/77dbc460-05a9-481d-8743-2c3bd242e23b" />

### Operating system

NixOS Linux

### Application version

Element version: 1.12.14 Crypto version: Rust SDK 0.16.0 (1ac734e), Vodozemac 0.9.0

### How did you install the app?

nixpkgs

### Homeserver

element.io

### Will you send logs?

No

**Comment by Half-Shot** [comment]
Huh, I have in all my years never seen this one pop up. It's apparently linked to EchoChambers so in very specific situations we might show this toast if you happen to fail to save a setting I believe.

Given this was added 5 years ago and not really adopted in the codebase, we might want to just rip it out at this point.

**Comment by Half-Shot** [comment]
Marking as uncommon as triggering this error needs your notification volume setting to fail to be set, so it's super infrequent.
